src/transformers/models/mmbt/modeling_mmbt.py

```diff
@@ -94,9 +94,9 @@ def forward(self, input_modal, start_token=None, end_token=None, position_ids=No
         config ([`MMBTConfig`]): Model configuration class with all the parameters of the model.
             Initializing with a config file does not load the weights associated with the model, only the
             configuration.
-        transformer (:class: *~nn.Module*): A text transformer that is used by MMBT.
+        transformer ([`~nn.Module`]): A text transformer that is used by MMBT.
```

Comment:
```suggestion
        transformer (`nn.Module`): A text transformer that is used by MMBT.
```
Won't be resolved since it's external, so need for the brackets.